main.tf

```diff
@@ -116,22 +125,37 @@ resource "aws_instance" "default" {
   tags = "${module.label.tags}"
 }
 
+##
+## Create keypair if one isn't provided
+##
+
+module "ssh_key_pair" {
+  source                = "git::https://github.com/cloudposse/terraform-aws-key-pair.git?ref=master"
+  namespace             = "${var.namespace}"
+  stage                 = "${var.stage}"
+  name                  = "${var.name}"
+  ssh_public_key_path   = "${path.cwd}"
```

Comment:
This should definitely be an input (or can default to `./`). Also, using `path.cwd` may not portable across different workstations which may use different paths.